PKCS12.xs

```diff
@@ -992,6 +996,18 @@ new(class)
   OUTPUT:
   RETVAL
 
+IV legacy_support(class)
+  SV *class;
+
+  CODE:
+  RETVAL = 0;
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L
```

Comment:
If OPENSSL_VERSION_NUMBER < 0x30000000 and not libressl it should like default to support for the legacy protocols.